.envsample

```diff
@@ -0,0 +1,9 @@
+#PORT
+PORT=
+
+#DATABASE VARIABLE
+
+USERNAME=
+PASSWORD=
+DATABASE=
+HOST=
```

Comment:
variable names are diff from what is used

Reply:
Updated